src/pacer.js

```diff
@@ -586,9 +589,17 @@ let PACER = {
     //  - doc_number
     //  - att_number
     
+    let audio_string = $('td:contains(AUDIO)').text();
```

Comment:
We should refactor this method. I think we can use short-circuit evaluation to avoid repeating the regex search. 

We can do something like this:

```javascript
let audio_string = $('td:contains(AUDIO)').text();
let image_string = $('td:contains(Image)').text();
let transcript_string = $('td:contains(TRANSCRIPT)').text();

let receipt_description = image_string || audio_string || transcript_string;
let regex = /(\d+)-(\d+)/;
let matches = regex.exec(receipt_description);
if (!matches) {
  return null;
}
```

Reply:
applied as suggested

Reply:
thanks!